examples/experimental/onnx/semantic_segmentation/README.md

```diff
@@ -0,0 +1,62 @@
+# Semantic Segmentation sample
+
+This sample shows an example of quantization of segmentation models. The used datasets are CamVid, Mapillary.
+
+## Install
+
+Please, install the requirements for ONNX Post-Training Quantization of NNCF.
+
+Install requirements
+
+```
+pip install -r <nncf dir>/nncf/experimental/onnx/requirements.txt
+pip install -r <nncf dir>/examples/experimental/onnx/requirements.txt
+```
+
+## Getting the quantized model
+
+To run post-training quantization on your model you can use the following command.
+
+```
+python onnx_ptq_segmentation.py -m <ONNX model path> -o <quantized ONNX model path> --dataset_name <CamVid or Mapillary> --data <dataset path>
+```
+
+Also, you could specify some options of quantization, please, take a look at the argument description by using the
+command:
+
+```
+python onnx_ptq_classification.py --help
+```
+
+## Measuring the accuracy of the original and quantized models
+
+If you would like to compare the accuracy of the original model and quantized one, you could
```

Comment:
I wonder why do we need AC at all if we have a correct implementation of data loader. We can create a simple validation function and use skylearn for metric computation (e.g. https://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_score.html). In this case, it would be an end-to-end example.